packages/fx-core/src/plugins/resource/bot/errors.ts

```diff
@@ -34,12 +37,45 @@ export class PluginError extends Error {
     this.errorType = type;
     this.innerError = innerError;
     this.helpLink = helpLink;
+    this.inferFromInnerError();
     Object.setPrototypeOf(this, PluginError.prototype);
   }
 
   genMessage(): string {
     return `${this.message} Suggestions: ${this.suggestions.join(" ")}`;
   }
+
+  inferFromInnerError() {
+    if (!this.innerError) return;
+
+    const errorCode = this.innerError.response?.data?.error?.code;
+    const helpLink = GraphErrorCodes.get(errorCode);
```

Comment:
Looks not consistent for me.
1. This line should only applies for Graph Errors, but the class name is PluginError.
2. Get out a helpLink by GraphErrorCodes.get? Not good from readability.

Reply:
Yes it only has graph help link, for other's error, the helpLink =GraphErrorCodes.get(errorCode) will be undefined, and use the parameter's helpLink. The GraphError help link is special for it needs to show different helpLinks according to ErrorType.